src/csharp/Grpc.Core/GrpcEnvironment.cs

```diff
@@ -50,15 +50,15 @@ public class GrpcEnvironment
         static int requestCallContextPoolThreadLocalCapacity = DefaultRequestCallContextPoolThreadLocalCapacity;
         static readonly HashSet<Channel> registeredChannels = new HashSet<Channel>();
         static readonly HashSet<Server> registeredServers = new HashSet<Server>();
-        static readonly AtomicCounter nativeInitCounter = new AtomicCounter();
+        static long nativeInitCounter = AtomicCounter.Create();
 
         static ILogger logger = new LogLevelFilterLogger(new ConsoleLogger(), LogLevel.Off, true);
 
         readonly IObjectPool<BatchContextSafeHandle> batchContextPool;
         readonly IObjectPool<RequestCallContextSafeHandle> requestCallContextPool;
         readonly GrpcThreadPool threadPool;
         readonly DebugStats debugStats = new DebugStats();
-        readonly AtomicCounter cqPickerCounter = new AtomicCounter();
```

Comment:
GrpcEnvironment is created even more rarely than the Channel, so I'm not sure this is worth doing either.